src/util/input/prompt-bool.ts

```diff
@@ -50,12 +50,15 @@ export default async function promptBool(label: string, options: Options = {}) {
         stdout.write(`\n`);
         resolve(true);
       } else if (data[0].toLowerCase() === noChar) {
+        stdout.write(`\n`);
```

Comment:
this change add a new-line after any response to promptBool, to fix the missing '\n' when promptBool get an negative input ("n"/"N"/Enter/Abort).

Before:

    ~/git » now init koa
    > Did you mean nodejs-koa? [y|N] > No changes made.

After:

    ~/git » now init koa
    > Did you mean nodejs-koa? [y|N]
    > No changes made.

But I'm not sure if this change would cause some unexpected output in other use cases. @rauchg @leo Could you confirm if this fine?